Core/src/org/sleuthkit/autopsy/timeline/ui/listvew/datamodel/ListViewModel.java

```diff
@@ -106,11 +106,21 @@ public ListViewModel(FilteredEventsModel eventsModel) {
                 List<Long> eventIDs = unGroupConcat(resultSet.getString("eventIDs"), Long::valueOf);
                 List<EventType> eventTypes = unGroupConcat(resultSet.getString("eventTypes"),
                         typesString -> eventManager.getEventType(Integer.valueOf(typesString)).orElseThrow(() -> new TskCoreException("Error mapping event type id " + typesString + ".S")));
+                
                 Map<EventType, Long> eventMap = new HashMap<>();
-                for (int i = 0; i < eventIDs.size(); i++) {
-                    eventMap.put(eventTypes.get(i), eventIDs.get(i));
+                
+                if(hasFileTypeEvents(eventTypes)) {
```

Comment:
We should add a comment here to the effect of:
"We want to merge together file sub-type events that are at the same time, but create individual events for other sub types of events (such as web history and web download)".